### PR TITLE
Tiled gallery: Move margin constant to constants

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

* Moves a constant into shared constants file

#### Testing instructions

* No regressions in mosaic layout

### CODE REVIEW

File: client/gutenberg/extensions/tiled-gallery/constants.js [hunk 1]
```diff
@@ -30,4 +30,5 @@ export const LAYOUT_STYLES = [
 	*/
 ];
 export const MAX_COLUMNS = 20;
+export const MOSAIC_MARGIN = 4;
```
Comment (simison) [inline client/gutenberg/extensions/tiled-gallery/constants.js]: Wouldn't this be the margin for all the layouts, including columns?

Comment (sirreal) [reply client/gutenberg/extensions/tiled-gallery/constants.js]: I suppose so

Comment (sirreal) [reply client/gutenberg/extensions/tiled-gallery/constants.js]: I think it should be `GUTTER` as well.

**Comment by sirreal** [comment]
Updated to use `GUTTER_WIDTH`

**Pull request merged**